guidelines/monai-application-package.md

```diff
@@ -1,498 +1,650 @@
+| `output.format`                 | **Yes**                                             | N/A            | object  | object                     | Details about the format of the outputs produced by the application.                                                                                               |
+| `output.path`                   | No                                                  | output/        | string  | relative file-system path  | Folder path relative to the working directory to which the application will write outputs.                                                                         |
+| `sdk`                           | No                                                  | N/A            | string  | string                     | SDK used for the Application.                                                                                                                                      |
+| `sdkVersion`                    | No                                                  | 0.0.0          | string  | semantic version           | Version of the SDK used the Application.                                                                                                                           |
+| `timeout`                       | No                                                  | 0              | integer | number                     | The maximum number of seconds the application should execute before being timed out and terminated. Recommended for a single batch/execution type of applications. |
+| `version`                       | No                                                  | 0.0.0          | string  | semantic version           | Version of the Application.                                                                                                                                        |
+| `workingDirectory`              | No                                                  | /var/holoscan/ | string  | absolute file-system path  | Folder, or directory, in which the application will be executed.                                                                                                   |
+
+The Application Manifest file provides information about the MAP's Application.
+
+- The Application Manifest MUST define the type of the containerized application (`/etc/holoscan/app.json#type`).
+
+  - Type SHALL have the value of either `service` or `application.`
+
+- The Application Manifest MUST define the command used to run the Application (`/etc/holoscan/app.json#command`).
+
+  - When not provided, the MAP will be considered invalid and not runnable by MONAI Deploy Application Runner and hosting services that comply to MAP specification.
```

Comment:
Shouldn't be specified that if any requirement marked as "MUST" is missing then the MAP is invalid, not just this requirement? A MAP without the type specification, the previous bullet point, shouldn't be valid either I think.

Reply:
The statement in the Requirement covers that. We can remove this to avoid confusion!

Reply:
Remove statement and updated `Requirements` section!